business_requirement_deliverable_project_task_mandatory/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================================================
+Business Requirement Deliverable Project Task Mandatory
+=======================================================
+
+This module extends the functionality of Project Category
+Business Requirement Project and allow to set a Category
+that will make required have all task linked to a Business Requirement.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Project Category
+#. Select one Category and check br_required
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/Elico-Corp/odoo-addons/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Eric Caudal <eric.caudal@elico-corp.com>
+* Victor Martin <victor.martin@elico-corp.com>
+
+Maintainer
```

Comment:
Actually the project category doesnot work fine AFAIK and we should switch to the new one asap